doc/frontend_devel.md

```diff
@@ -100,6 +100,11 @@ settings that affect the frontend.
     - value: swe
       label: Svenska
     ```
+- __logo__ - Object. Specify site-specific logos.
+  - __korp__ - String. HTML content for the Korp logo to the left of the corpus chooser. Default: [plain Korp logo](../app/img/korp.svg).
+  - __organization__ - String. HTML content for the organization logo(s) on the top right of the Korp window. Default: empty.
+  - __chooser_right__ - String. HTML content for a logo to the right of the corpus chooser: Default: empty.
+  The HTML content can refer to image files in the `app/img/` directory of the configuration as `img/`_file_, and to the [plain Korp logo](../app/img/korp.svg) as `img/korp.svg`.
```

Comment:
Here, could we add something like

> Note that custom images may be over-cached by browsers. To avoid this, consider manually renaming images when they are changed.

And, if you agree with removing the `korp.svg` copy directive, remove the phrase about that here.